sql3/planner/compilebulkinsert.go

```diff
@@ -203,6 +203,14 @@ func (p *ExecutionPlanner) analyzeBulkInsertStatement(stmt *parser.BulkInsertSta
 				return sql3.NewErrIntegerLiteral(im.MapExpr.Pos().Line, im.MapExpr.Pos().Column)
 			}
 		}
+	case "PARQUET":
+		// for paquet the map expressions need to be string values
+		// that represent the offsets in the source file
+		for _, im := range stmt.MapList {
+			if !(im.MapExpr.IsLiteral() && typeIsString(im.MapExpr.DataType())) {
+				return sql3.NewErrIntegerLiteral(im.MapExpr.Pos().Line, im.MapExpr.Pos().Column)
```

Comment:
I don't quite understand the "must be string" logic, but does that mean this error should be `NewErrStringLiteral`?

Reply:
good ole copy paste get's me every time